oteapi/models/datacacheconfig.py

```diff
@@ -1,10 +1,12 @@
 """Pydantic DataCache Configuration Data Model."""
 from pathlib import Path
 
-from pydantic import BaseModel, Field
+from pydantic import Field
 
+from oteapi.models.genericconfig import GenericConfig
 
-class DataCacheConfig(BaseModel):
+
+class DataCacheConfig(GenericConfig):
```

Comment:
```suggestion
class DataCacheConfig(AttrDict):
```
It's a bit misleading, but this configuration class is actually not for a strategy, but for a strategy configuration's `configuration` attribute......

Reply:
If it will never be used as anything else, then ok. I've made the change and updated the docstring to mention this.